app/Http/Controllers/ArticleControllerOwner.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Article;
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Contracts\View\Factory;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\Request;
+
+class ArticleControllerOwner extends Controller
```

Comment:
一応、細かいけどControllerはControllerやから、最後がControllerで終わった方がいいと思う。

```suggestion
class ArticleOwnerController extends Controller
```